tests/unit/config_test.py

```diff
@@ -40,6 +40,50 @@ def test_config_validation(self):
         config.make_service_dict('foo', {'ports': ['8000']})
 
 
+class BuildPathTest(unittest.TestCase):
+    def test_nonexistent_path(self):
+        options = {'build': 'nonexistent.path'}
+        self.assertRaises(
+            config.ConfigurationError,
+            lambda: config.make_service_dict('foo', options, 'tests/fixtures/build-path'),
+        )
+
+    def test_relative_path(self):
+        relative_build_path = '../build-ctx/'
+        resolved_build_path = '/code/tests/fixtures/build-ctx'
```

Comment:
That should work fine, but as far as I can tell it's equivalent to `os.path.abspath('tests/fixtures/build-ctx')`, which is simpler.